qcodes/tests/dataset/dond/test_do0d.py

```diff
@@ -20,7 +20,7 @@
 @pytest.mark.usefixtures("plot_close", "experiment")
 @pytest.mark.parametrize("period", [None, 1])
 @pytest.mark.parametrize("plot", [None, True, False])
-@pytest.mark.parametrize("plot_config", [None, True, False])
+@pytest.mark.parametrize("plot_config", [True, None, False])
```

Comment:
This test was leaking config but if done like this it will leak into the next variant here which causes a test failure